pkg/build/kube/bazelbuildbits.go

```diff
@@ -56,74 +60,114 @@ func (b *BazelBuildBits) Build() error {
 	// make sure we cd back when done
 	defer os.Chdir(cwd)
 
+	// TODO(bentheelder): we assume the host arch, but cross compiling should
+	// be possible now
```

Comment:
"we assume the host arch" won't this be invalid on OSX?

Reply:
the arch is still amd64 😅 
also kubelet's CGO binary was not buildable natively on mac with bazel unless you did some fun crosstool stuff AFAIK. I don't think this is supported yet.